.github/workflows/svg-compare.yml

```diff
@@ -35,13 +35,20 @@ jobs:
                   path: owid-grapher-svgs
                   token: ${{ env.TOKEN }}
 
-            # Switch to or create a branch on that repo that matches the branch name we are on in the owid-grapher repo
+            # Create a branch on that repo that matches the branch name we are on in the owid-grapher repo
             # but only do this if we are not on master in owid-graphers (in this case we want to commit and push on master
             # in owid-grapher-svgs as well)
             - name: create owid-grapher-svgs local branch
               if: ${{ env.PUSH_BRANCH_NAME != 'master'}}
               working-directory: owid-grapher-svgs
-              run: git branch ${{ env.PUSH_BRANCH_NAME }} && git checkout ${{ env.PUSH_BRANCH_NAME }}
+              continue-on-error: true
```

Comment:
If both actions commit to the same branch in `owid-grapher-svgs`, then creating a branch should be a no-op if the branch already exists. `git branch <b>` fails with exit code 1 if `<b>` exists, that's why I set `continue-on-error` to true. I wasn't sure whether the next line (the checkout) would run if the previous line failed. To be on the safe side, the branch is checked out in a separate step.